Content.Client/Atmos/UI/SpaceHeaterWindow.xaml.cs

```diff
@@ -0,0 +1,71 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Content.Shared.Atmos.Piping.Portable.Components;
+using Content.Shared.Atmos;
+
+namespace Content.Client.Atmos.UI;
+
+/// <summary>
+/// Client-side UI used to control a space heater.
+/// </summary>
+[GenerateTypedNameReferences]
+public sealed partial class SpaceHeaterWindow : DefaultWindow
+{
+    // To account for a minimum delta temperature for atmos equalization to trigger we use a fixed step for target temperature increment/decrement
+    public int TemperatureChangeDelta = 5;
+    public bool Active = false;
+    public float MinTemp = 0.0f;
+    public float MaxTemp = 0.0f;
```

Comment:
These are all the defaults, and do not need to be explicitly specified, unless the defaults are non-obvious and deserve to be explicitly specified.